app/views/main/index.html.erb

```diff
@@ -1,9 +1,7 @@
 <section class="grid-container">
-  <div class="top-news"> <%= render 'partials/toparticle' %></div>
-  <div class="news-feed py-2 px-4"><%= render 'partials/apifeed' %></div>
+  <div class="breaking-news"> <%= render 'partials/toparticle' %></div>
```

Comment:
I had to change breaking news to top news to pass tests are the tests still passing